src/device-lister.js

```diff
@@ -210,3 +211,8 @@ export default class DeviceLister extends EventEmitter {
         return deviceMap;
     }
 }
+
+module.exports = {
+    DeviceLister,
+    ErrorCodes,
+};
```

Comment:
This will unfortunately break existing code, which expects the `DeviceLister` to be exposed using `default export`.

An alternative approach which keeps backwards compatibility is to add `ErrorCodes` as a property on the `DeviceLister`, like:

```
DeviceLister.ErrorCodes = ErrorCodes;
export default DeviceLister;
```

Code that uses the device lister can then refer to the error codes with e.g.:

```
DeviceLister.ErrorCodes.RECEIVED_NEITHER_SNO_NOR_ERROR
```

Reply:
I thought this might be an issue if dependants where to use a newer version of nrf-device-lister-js, without refactoring to use the new changes. If adding ErrorCodes to DeviceListener is an acceptable solution, then I will do that.